context/package.json

```diff
@@ -7,6 +7,7 @@
     "@material-ui/core": "^4.9.4",
     "@material-ui/icons": "^4.9.1",
     "@material-ui/lab": "^4.0.0-alpha.45",
+    "date-fns": "^2.14.0",
```

Comment:
Slight preference to use the YYYY-MM-DD format straight from the index... Unless there's been a directive to go with a particular format, in which case need to make sure it gets applied everywhere.

Reply:
I'd like to keep this parallel to the design. I can file an issue to update once the design is.